src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/migration/MigrationClient.kt

```diff
@@ -23,26 +21,30 @@ import java.time.LocalTime
 @Component
 @Deprecated(message = "Can be removed when migration is completed")
 class MigrationClient(
-  private val prisonerSearchApiWebClient: WebClient,
   private val locationsInsidePrisonApiWebClient: WebClient,
   private val whereaboutsApiWebClient: WebClient,
   private val nomisMappingApiWebClient: WebClient,
+  private val prisonApiWebClient: WebClient,
 ) {
   companion object {
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
-  fun getPrisoner(bookingId: Long): Prisoner? =
-    prisonerSearchApiWebClient.post()
-      .uri("/prisoner-search/booking-ids")
-      .bodyValue(BookingIds(listOf(bookingId)))
+  // Unsure how effective this will be as requests will spread out over the pods unless we reduce number of pods.
```

Comment:
Adding caching though not really sure how effective this will be due to events being handle over N pods.